### ISSUE TITLE
tuple "index" is not a known memeber

### ISSUE THREAD

**Issue by jerryc05** [Microsoft/pyright#47]
**Describe the bug**
```
a=('1','2')
a.index('1')
```
A red underscore is shown under "index" method

**To Reproduce**
type
```
a=('1','2')
a.index('1')
```

**Expected behavior**
no red underscore

**Screenshots or Code**
```
a=('1','2')
a.index('1')
```

**VS Code extension or command-line**
VS Code extension v1.0.8

**Additional context**

**Comment by erictraut** [comment]
Thanks for the bug report. I've found and fixed the cause of the bug. The fix will be in the next published version.

**Comment by erictraut** [comment]
This is fixed in 1.0.9, which I just published. Thanks again for the report.
